src/wp-includes/class-wp-image-editor-imagick.php

```diff
@@ -21,6 +21,15 @@ class WP_Image_Editor_Imagick extends WP_Image_Editor {
 	 */
 	protected $image;
 
+	/**
+     * Stores the information whether the image is indexed-color encoded
+     *
+     * @since 4.7
+     * @access protected
+     * @var bool
+     */
```

Comment:
```suggestion
	/**
	 * Stores the information whether the image is indexed-color encoded
	 *
	 * @since 4.7
	 * @access protected
	 * @var bool
	 */
```